README.md

```diff
@@ -191,18 +191,23 @@ $ npm install --save drivelist
 Documentation
 -------------
 
+
+* [drivelist](#module_drivelist)
+    * [.list(callback)](#module_drivelist.list)
+    * [.get(device, callback)](#module_drivelist.get)
+
 <a name="module_drivelist.list"></a>
 
 ### drivelist.list(callback)
-**Kind**: static method of <code>[drivelist](#module_drivelist)</code>
-**Summary**: List available drives
-**Access:** public
+**Kind**: static method of <code>[drivelist](#module_drivelist)</code>  
+**Summary**: List available drives  
+**Access:** public  
```

Comment:
It seems to come from the JSDoc to markdown module. I could manually edit it, but then they will re-appear the next time we re-generate it.